@kiva/kv-components/vue/.storybook/preview.js

```diff
@@ -1,17 +1,15 @@
 import './tailwind.css';
+import {
+	setup
+} from '@storybook/vue3';
 import { addons } from '@storybook/preview-api';
 import KvThemeProvider from '../KvThemeProvider.vue';
 import { defaultTheme, darkTheme } from '@kiva/kv-tokens/configs/kivaColors.cjs';
-import Vue from 'vue';
-import VueCompositionApi from '@vue/composition-api';
-import VueRouter from 'vue-router';
 
-// Add vue composition api
-Vue.use(VueCompositionApi);
+setup((app) => {
```

Comment:
Will code be added here? Or do we just need this empty `setup` in general?

Reply:
I think we will need to add some things here. Like global components, vue router, etc.